### ISSUE TITLE
Unnecessary scroll bar

### ISSUE THREAD

**Issue by Pantkowsky** [corradio/electricitymap#351]
The current interface (when using chrome) looks like this:

![untitled](https://cloud.githubusercontent.com/assets/9194725/22712048/b3608b48-ed83-11e6-9367-27824e6ff899.png)

However, the scrollbar is not really necessary, as there is nothing to scroll to. It therefore appears to be a bug.

The situation for IE looks even more confusing, as scrolling is enabled outside of the boundaries of the map:

![untitled](https://cloud.githubusercontent.com/assets/9194725/22712173/19a7b110-ed84-11e6-96f1-e6d7548e859c.png)

![untitled](https://cloud.githubusercontent.com/assets/9194725/22712222/51c2015e-ed84-11e6-8b57-baa6b766cf50.png)

![untitled](https://cloud.githubusercontent.com/assets/9194725/22712199/378ffc96-ed84-11e6-9c38-3a0d2e49f737.png)

**Comment by corradio** [comment]
So this is happening on Windows 10 right?
@nikkozzblu do you have the same errors on IE/Chrome?

**Comment by nikkozzblu** [comment]
Correct, thats why i was refering to it as «tab» look like the early html tab tag :)

Nico Juguet

-------- Mensaje original --------
Asunto: Re: [corradio/electricitymap] Unnecessary scroll bar (#351)
De: Olivier Corradi
Para: corradio/electricitymap
CC: nikkozzblu ,Mention


So this is happening on Windows 10 right?
@nikkozzblu<https://github.com/nikkozzblu> do you have the same errors on IE/Chrome?

**Comment by nikkozzblu** [comment]
Overflow-y hidden seems to do the trick, while you dont need overflow

Nico Juguet

-------- Mensaje original --------
Asunto: Re: [corradio/electricitymap] Unnecessary scroll bar (#351)
De: Olivier Corradi
Para: corradio/electricitymap
CC: nikkozzblu ,Mention


So this is happening on Windows 10 right?
@nikkozzblu<https://github.com/nikkozzblu> do you have the same errors on IE/Chrome?

**Comment by corradio** [comment]
@nikkozzblu can you take a look asap? Most of our users are on Windows and I can't reproduce.
